File: Scala/alg05.scala
/* ALG05 */
import scala.io.StdIn
import scala.math.pow
object Estudo extends App {
  var X: Int = 0
  var Y: Int = 0
  var Z: Int = 0
  X = scala.io.StdIn.readInt()
  Y = scala.io.StdIn.readInt()
  Z = pow(X, 2).toInt + pow(Y, 2).toInt
  println(Z)
}

File: C#/alg05.cs
using System;
class ALG05
{
    static void Main()
    {
        int X;
        int Y;
        int Z;
        int.TryParse(Console.ReadLine(), out X);
        int.TryParse(Console.ReadLine(), out Y);
        Z = (int)Math.Pow(X, 2) + (int)Math.Pow(Y, 2);
        Console.WriteLine(Z);
    }
}
